### PR TITLE
COMMERCE-6098

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/DDMFormHandler.js [hunk 1]
```diff
@@ -70,11 +70,4 @@ class DDMFormHandler {
 	}
 }
 
-Liferay.component(
-	'DDMFormHandler',
-	(() => ({
-		attach: (configuration) => new DDMFormHandler(configuration),
-	}))()
-);
-
```
Comment (gianmarcobrunialti) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/DDMFormHandler.js:79]: This was never really used and still is not. Removed.

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Review by ivicac — APPROVED** [review]

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:forward

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:stop

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:forward

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:stop

**Comment by gianmarcobrunialti** [comment]
ci:forward

**Comment by ivicac** [comment]
Merged by https://github.com/brianchandotcom/liferay-portal/pull/100138

**Pull request closed without merging**